csharp/src/Ice/Util.cs

```diff
@@ -77,7 +77,7 @@ public sealed class Util
     /// <returns>A new empty property set.</returns>
     public static Properties createProperties()
     {
-        return new PropertiesI();
+        return new Properties();
```

Comment:
The logic is we provide this method for backwards compatibility?

It's naturally simpler to call the ctor directly.